client/src/app/styles/_variables.scss

```diff
@@ -16,3 +16,19 @@ $l-font: 18px;
 $xl-font: 20px;
 $xxl-font: 24px;
 $xxxl-font: 32px;
+
+// margins
+$s-margin: 4px;
+$m-margin: 8px;
+$l-margin: 16px;
+$xl-margin: 32px;
```

Comment:
why so huge values, we might rather need some smaller one like 2px

Reply:
on a meeting, in the last week, we said we gonna have margins multiplied by 4px so that's why I did like so :)
 so, what margins we should have?

Reply:
it is ok but do we need constants for margins like 64 and 128px? How often will we use them?